spec/requests/api/v1/users/update_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'User account update', type: :request do
+  let(:user) { create(:user) }
+  let(:headers) { user.create_new_auth_token }
+  let(:user_params) { {} }
+  subject do
```

Comment:
Leave empty line in between these two:
https://github.com/rubocop/rspec-style-guide#empty-line-after-let